content/05.project-management.md

```diff
@@ -81,7 +81,7 @@ You can then generate a digital object identifier (DOI) for that release using a
 
 Sharing your workflow code with collaborators, peer reviewers, and scientists seeking to use a similar method can foster discussion and review of your analysis.
 Sticking to a clear documentation strategy, using a version control system, and packaging your code in notebooks or as a workflow prepare them to be easily shared with others.
-To go one step further, you can package your code with a tool like Binder, Whole Tale, or Shiny apps, which run the code on cloud computers in environments identical to those in which the original computation was performed (**Figure @fig:nb_figure**, **Figure @fig:interactiveviz**) [@doi:10.25080/Majora-4af1f417-011; @doi:10.1016/j.future.2017.12.029].
+To go one step further, you can package your code with tools like Binder and Whole Tale (CTB: reprozip, other such things?), or use Shiny apps. These approaches let others run the code on cloud computers in environments identical to those in which the original computation was performed (**Figure @fig:nb_figure**, **Figure @fig:interactiveviz**) [@doi:10.25080/Majora-4af1f417-011; @doi:10.1016/j.future.2017.12.029].
```

Comment:
```suggestion
To go one step further, you can package your code with tools like Binder, ReproZip, or Whole Tale, or make interactive visualizations with tools like Shiny apps or Plotly. These approaches let others run the code on cloud computers in environments identical to those in which the original computation was performed (**Figure @fig:nb_figure**, **Figure @fig:interactiveviz**) [@doi:10.25080/Majora-4af1f417-011; @doi:10.1016/j.future.2017.12.029; @https://www.usenix.org/conference/tapp13/reprozip-using-provenance-support-computational-reproducibilitythe].
```

Reply:
Hi! If you don't mind, could you use [our preferred citation](https://github.com/VIDA-NYU/reprozip/blob/master/CITATION.txt)? This is an outdated reference about an older version of the system, we're not sure why it keeps popping up.

Thanks!

Reply:
done -- Thank you!!